src/renderer/cursor_renderer/mod.rs

```diff
@@ -20,8 +20,9 @@ const DEFAULT_CELL_PERCENTAGE: f32 = 1.0 / 8.0;
 
 const STANDARD_CORNERS: &[(f32, f32); 4] = &[(-0.5, -0.5), (0.5, -0.5), (0.5, 0.5), (-0.5, 0.5)];
 
-#[setting_prefix = "cursor"]
-#[derive(Clone, SettingGroup)]
+#[derive(SettingGroup)]
+#[setting_prefix = "keyboard"]
```

Comment:
correct me if I'm wrong, but I believe this represents a breaking change

Reply:
It behaves the same when I run it. Clippy was complaining about it and saying that the previous line would be counted as an error in the future.

Reply:
to be specific, setting_prefix is changed from cursor to keyboard here. Thats what I was pointing out.